script/itest

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+# Wrapper for running integration tests in docker.
+
+./script/dock pkg $(pwd)/bin
```

Comment:
This seems to assume that the `gh-ost` binary in `$(pwd)/bin` is up to date and built from the current branch/changes. This may not be the case.